spring-security-oauth/src/main/java/org/springframework/security/oauth/common/OAuthParameter.java

```diff
@@ -17,11 +17,12 @@
 package org.springframework.security.oauth.common;
 
 /**
- * Enumeration for consumer parameters.
+ * OAuth parameters.
  *
  * @author Ryan Heaton
+ * @author <a rel="author" href="http://autayeu.com/">Aliaksandr Autayeu</a>
  */
-public enum OAuthConsumerParameter {
```

Comment:
I know that this enum was here before your chances but, I'm not quite sure why they decided to make this an Enum if they are only making reference to this enum class to get their constants, I can't find any reference in the code where they are storing an ``OAuthConsumerParameter`` or a ``OAuthParameter`` type in any place, it seems that all the places in the project where they made reference to this enum is to retrieve one of it's enum instances and made a `Enum.toString()` call, why instead of having the enum we don't just made this a constants class, that way we can call the public constant and we can save the `.toString()` call in the code of those places we made reference to this.